packages/aws-cdk/lib/init-templates/app/python/app.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+from aws_cdk import cdk
+
+from hello.hello_stack import PyStack
```

Comment:
We have some templating foo that you can use to generate a name for this stack and files based on the directory name in which the project was initialized (see typescript as an example). Otherwise, this means users will need to modify this after they've initialized the app and ideally they shouldn't have to.

Reply:
Latest commit uses templating.